website/_posts/blogs/2016-06-29-myriad-0.2.0-release-announcement.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: big_card_for_md
+title: Apache Myriad 0.2.0 (Incubating) Released
+author: <a href="https://github.com/darinj">Darin Johnson</a> (Apache Myriad committer, 0.2.0 Release Manager)
+date:   2016-06-29 00:00:00
+categories: blogs
+---
+
+##### {{page.title}} 
+
+On behalf of the entire Myriad Team, I’m happy to announce the release of Apache Myriad 0.2.0 (incubating).  This is the 
+second release of Apache Myriad.  Among the features of the 0.2.0 release are increased reliability of Fine Grained Scaling, support for running the Node 
+Manager and Auxiliary Services (Jobhistory Server, etc) in Docker Containers, and better distribution management of the 
+Hadoop configuration and binaries.  This release brings Myriad much closer to being production quality.
+
+In addition to new features, we also welcome our newest committer Yuliya Feldman (MapR) and a new contributor Zachary Jaffee 
+(Case Western Reserve University).  
+
+Many thanks to the following individuals who contributed code/documentation to the Myriad project. This release wouldn’t 
+have been possible without their effort.
+
+* Aashreya Ravi Shankar
+* Adam B
+* Brandon Gulla
+* Darin Johnson
+* Jim Klucar
+* Kannan Rajah
+* Ken Sipe
+* Kris Chi
+* Larry Edelstein
+* Mohit Soni
+* Naveen Manivannan
+* Paul Read
+* Ruth Harris
+* Santosh Marella
+* Sarjeet Singh
+* Shingo Omura
+* Suneel Marthi
+* Swapnil Daingade
+* Yuliya Feldman
+* Zachary Jaffee
+* Zhongyue Luo
+ 
+This is the second of many Apache Myriad releases. We've already started merging new pull requests for the next release, 
+where we plan to increase tackle YARN Security and adding additional Mesos features into Myriad such as the ability to use 
```

Comment:
"we plan to add support for YARN Security and advanced Mesos features like reservations/volumes and oversubscription."